state-chain/pallets/cf-governance/src/migrations.rs

```diff
@@ -1,5 +1,9 @@
 use crate::Pallet;
+use cf_runtime_upgrade_utilities::{migration_template, VersionedMigration};
 
-use cf_runtime_upgrade_utilities::{migration_template::Migration, VersionedMigration};
+pub mod v2;
 
-pub type PalletMigration<T> = VersionedMigration<Pallet<T>, Migration<T>, 0, 1>;
+pub type PalletMigration<T> = (
+	VersionedMigration<Pallet<T>, v2::Migration<T>, 1, 2>,
+	VersionedMigration<Pallet<T>, migration_template::Migration<T>, 2, 3>,
```

Comment:
Seems wrong?

Reply:
`VersionedMigration<Pallet<T>, migration_template::Migration<T>, 2, 3>,`

Reply:
It's intentional. The migration is required to increment the reference counter for governance accounts. If you mean the presence of the migration_template, this is just a convention, makes it more convenient to add a new migration.

Reply:
Like the Pallet version is 2 in the lib.rs, but we have what I assume is a blank migration to version 3?

Reply:
Yes - actually it's an `unimplemented!()` migration that won't be applied (because the pallet version is 2). And if you accidentally do use it, the upgrade checks will panic and tell you off. 

It means that when you delete an old migration, you can leave the template as a placeholder. Makes the diff nicer and makes it easier to add the 'next' migration.